api_utils/txtlib.py

```
@@ -0,0 +1,131 @@
+# ---
+from . import txtlib
+# temps = txtlib.extract_templates_and_params(text)
+# for temp in temps: name, namestrip, params, template = temp['name'], temp['namestrip'], temp['params'], temp['item']
+
+"""
+# from ..other_bots import printe
+
+import wikitextparser as wtp
+
+
+def extract_templates_and_params(text):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)